operator/webhook/introute/sync.py

```diff
@@ -388,9 +390,51 @@ def _new_deployment(parent):
 
     return deployment
 
+def _new_service(parent):
+    parent_metadata = parent["metadata"]
+
+    has_tls = _has_tls(parent)
+    scheme = _get_scheme(has_tls)
+    management_port = _get_management_port(has_tls)
+
+    service = {
+        "apiVersion": "v1",
+        "kind": "Service",
+        "metadata": {
+            "labels": {
+                "integration-route": parent_metadata["name"],
+                "prometheus-metrics-enabled": "true"
+            },
+            "name": parent_metadata["name"]
```

Comment:
Since the end user could potentially create other services for this route to handle incoming business logic traffic, could we give it a more specific name? Maybe `<route-name>-actuator`?

Reply:
Fixed